CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 * Add `aspect_ratio` ui for gallery and document blocks
 * Use `@triniti/schemas` v1.1.6
 * Create `VideoAssetPickerField` and use for video `mezzanine_ref`.
+* Up/Down arrow key in blocks bug
```

Comment:
@nanetteCodes - please describe briefly what the bug with the `up/down`  is. Right now, it is unclear to me.

Reply:
@nanetteCodes put the name of the ticket

Reply:
@arunkarnati / @efjacobson I updated the changelog